src/ConfigProvider.php

```diff
@@ -0,0 +1,365 @@
+            Helper\Identity::class            => Helper\Service\IdentityFactory::class,
+            Helper\BasePath::class            => InvokableFactory::class,
+            Helper\Cycle::class               => InvokableFactory::class,
+            Helper\DeclareVars::class         => InvokableFactory::class,
+            Helper\Doctype::class             => Helper\Service\DoctypeFactory::class,
+            Helper\EscapeHtml::class          => InvokableFactory::class,
+            Helper\EscapeHtmlAttr::class      => InvokableFactory::class,
+            Helper\EscapeJs::class            => InvokableFactory::class,
+            Helper\EscapeCss::class           => InvokableFactory::class,
+            Helper\EscapeUrl::class           => InvokableFactory::class,
+            Helper\Gravatar::class            => InvokableFactory::class,
+            Helper\HtmlTag::class             => InvokableFactory::class,
+            Helper\HeadLink::class            => InvokableFactory::class,
+            Helper\HeadMeta::class            => InvokableFactory::class,
+            Helper\HeadScript::class          => InvokableFactory::class,
+            Helper\HeadStyle::class           => InvokableFactory::class,
+            Helper\HeadTitle::class           => InvokableFactory::class,
+            Helper\HtmlFlash::class           => InvokableFactory::class,
+            Helper\HtmlList::class            => InvokableFactory::class,
+            Helper\HtmlObject::class          => InvokableFactory::class,
+            Helper\HtmlPage::class            => InvokableFactory::class,
+            Helper\HtmlQuicktime::class       => InvokableFactory::class,
+            Helper\InlineScript::class        => InvokableFactory::class,
+            Helper\Json::class                => InvokableFactory::class,
+            Helper\Layout::class              => InvokableFactory::class,
+            Helper\PaginationControl::class   => InvokableFactory::class,
+            Helper\PartialLoop::class         => InvokableFactory::class,
+            Helper\Partial::class             => InvokableFactory::class,
+            Helper\Placeholder::class         => InvokableFactory::class,
+            Helper\RenderChildModel::class    => InvokableFactory::class,
+            Helper\RenderToPlaceholder::class => InvokableFactory::class,
+            Helper\ServerUrl::class           => InvokableFactory::class,
+            Helper\Url::class                 => InvokableFactory::class,
+            Helper\ViewModel::class           => InvokableFactory::class,
+        ];
+    }
+
+    /**
+     * Default helper aliases
+     *
+     * Most of these are present for legacy purposes, as v2 of the service
+     * manager normalized names when fetching services.
+     *
+     * @return array<string,string>|array<array-key, string>
+     */
+    public static function defaultViewHelperAliases(): array
```

Comment:
Please mark this as `@internal` since there should be no reason to use this outside of this repository.
This also enables us to remove it in the next major.

Therefore, adding `@deprecated` would be helpful so we can find it when preparing the next major.